src/main/resources/META-INF/resources/primefaces/jquery/jquery.ui.pfextensions.js

```diff
@@ -160,6 +160,13 @@
                 }, 0);
         }
     };
+
+    $.datepicker._base_disableDatepicker = $.datepicker._disableDatepicker;
+    $.datepicker._disableDatepicker = function(target) {
+        setTimeout( function() {
+        	$.datepicker._base_disableDatepicker(target);
+        }, 0);
+    }
```

Comment:
I would put a comment above this "GitHub #5374 Calender inline=true not styled correctly" and also update the file "extensions.txt" which is located in that same directory.